manifests/init.pp

```diff
@@ -364,20 +357,18 @@
   Optional[Integer[0,255]] $ttl                                         = undef,
   Optional[Enum['ykd', 'none']] $vsftype                                = undef,
   Boolean $package_corosync                                             = $corosync::params::package_corosync,
-  Boolean $package_crmsh                                                = $corosync::params::package_crmsh,
```

Comment:
so this is backwards incompatible, so we should drop RHEL 6 support before we release this

Reply:
`crmsh` is still a valid option; it's like using `ifconfig` instead of `ip`. AFAIK, there's no deprecation, just a "this stack is better" argument. cf. [this FAQ post](https://wiki.clusterlabs.org/wiki/FAQ#Should_I_choose_pcs_or_crmsh.3F).